CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 ## Enhancements
 
 - Added `NormalizeSchemas` configuration property to the Avro, Json and Protobuf serdes.
+- Upgraded `NJsonSchema` to v10.6.3
```

Comment:
1.9.3 is the current version. 

2.0.0 is going to be the next version, make a new title and put under that.